### ISSUE TITLE
Remove Justify option from WYSWYG

### ISSUE THREAD

**Issue by kevincrafts** [CuBoulder/tiamat-theme#241]
Justifying text can cause some accessibility issues so this option should be removed.
![image](https://user-images.githubusercontent.com/349713/219173204-4b0e3082-29ab-4ab6-a119-2619d799d51c.png)
